.github/workflows/ci.yaml

```
@@ -0,0 +1,72 @@
+name: Run CI
+
+# Run this workflow every time a new commit pushed to your repository
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+env:
+  IMAGE_NAME: open-forms-ext-token-exchange
```

```suggestion
  IMAGE_NAME: openformulieren/open-forms-ext-token-exchange
```
needs the user/org prefix to push it to docker hub - not that we'll do that (except for an image having _all_ extensions), but probably good to establish the pattern